test/integration/fixtures/options/bail-with-after.fixture.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+describe('suite1', function () {
+  it('should only display this error', function (done) {
```

Comment:
please remove the `done` parameter in this function and below; they aren't necessary for what these tests are proving.